zebROS_ws/src/ros_control_boilerplate/config/2022_compbot_base_jetson.yaml

```diff
@@ -67,7 +67,10 @@ hardware_interface:
        - {name: robot_code_ready_rio, local: false, type: ready} # Probably not really needed?
 
        - {name: babys_first_orchestra, type: orchestra, id: 1}
-
+       - {name: elevator_master, type: can_talon_fx, can_id: 2, local: true}
```

Comment:
Remove these - these motors aren't on the 2022 robot so it will error immediately on real hardware. Restore the 2022 config files back to their state on master